en_US/data-integration/timestream-for-influxdb.md

```diff
@@ -0,0 +1,412 @@
+
+For more details about connection requirements and security considerations, see the AWS documentation: [Connecting to an Amazon Timestream for InfluxDB DB instance](https://docs.aws.amazon.com/timestream/latest/developerguide/timestream-for-influx-db-connecting.html).
+
+#### Obtain InfluxDB Token, Organization, and Bucket
+
+Token and credential retrieval depend on the **InfluxDB engine version** selected when creating your Timestream for InfluxDB instance.
+
+##### Access Influx UI for InfluxDB v2 DB Instance
+
+1. Open the **Influx UI** using the DB instance endpoint:
+
+   ```
+   https://<endpoint>:8086
+   ```
+
+   > If your DB instance is not publicly accessible, you must access the InfluxUI from a host within the same VPC (for example, via a bastion host or SSM port forwarding). See the [AWS documentation](https://docs.aws.amazon.com/timestream/latest/developerguide/timestream-for-influx-getting-started-creating-db-instance.html) for details.
+
+2. Log in using the master user credentials created with the DB instance.
+
+3. Generate or retrieve a personal access token with write permissions to the target bucket.
+
+   This is the token EMQX uses to authenticate with Timestream for InfluxDB.
+
+   ::: tip Note
+
+   Newly created tokens are shown only once. Be sure to copy and save them.
+
+   :::
+
+4. Confirm the **Organization** and **Bucket** values configured for your instance. These values must match exactly when configuring EMQX.
+
+Refer to the AWS official documentation for detailed instructions: [Access the InfluxDB UI](https://docs.aws.amazon.com/timestream/latest/developerguide/timestream-for-influx-getting-started-creating-db-instance.html#timestream-for-influx-getting-started-creating-db-instance-step-3).
+
+##### Retrieve Authentication Token for **InfluxDB v3** DB Instances
+
+InfluxDB v3 does not issue API tokens through the InfluxDB UI. Instead, AWS stores the authentication parameters, including the API token, in **AWS Secrets Manager** when the DB instance is created.
+
+1. Open the DB cluster details page in the Timestream console. Locate the field labeled: **Authentication properties Secret manager ARN**.
+
+   ![timestream_secret_arn](./assets/timestream_secret_arn.png)
+
+   This ARN points to the Secrets Manager entry that contains the credentials EMQX will use.
+
+2. Go to **AWS Secrets Manager** -> **Secrets** and search for the secret name shown (e.g., `READONLY-InfluxDB-auth-parameters-<cluster-id>`).
+
+3. Open the secret and switch to the **Plaintext** view to retrieve the secret contents.
+
+   ![timestream_secret_value](./assets/timestream_secret_value.png)
+
```

Comment:
I'm not familiar with maintaining influxdb instances, but I would assume that this master secret should be used to issue a new operator token for EMQX, much like what's describe in the above section for v2, in which the master credentials are used to generate an operator token for emqx.  I managed to use `influxdb3-explorer` to access the instance using the master credentials: https://docs.influxdata.com/influxdb3/explorer/